DependencyInjection/Configuration.php

```diff
@@ -33,8 +33,14 @@ public function __construct($debug)
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode    = $treeBuilder->root('doctrine');
+        $treeBuilder = new TreeBuilder('doctrine');
```

Comment:
Please use `use function` instead of `\`.

Reply:
not possible: ` "php": "^5.5.9|^7.0",` in composer.json

Reply:
Please, put the "new" statement in the "if" (with/without parms).
There is  other "new TreeBuilder()" in the file.

Reply:
@stof Using `\fn` directly violates our coding standard so it's invalid. If, due to PHP 5.5 compatibily, `use function` cannot be used, the function should be kept in _fallback_ mode.
This breaks build: https://travis-ci.org/doctrine/DoctrineBundle/jobs/443601176#L649